### DISCUSSION TITLE
Symfony Upgrade to 6.4 or 7.0

### DISCUSSION THREAD

**Discussion by hirakch** [pimcore/pimcore#16394] [Q&A / Support] [answered]
Currently, PIMCore 11.1.x is on Symfony 6.2. However Symfony has stopped support for 6.2 and released 6.4 LTS and 7.x stable version. Does PIMCore has any roadmap to upgrade Symfony versions in near future?

https://symfony.com/releases#maintained-symfony-branches

**Comment by twin-elements** [comment]
Run the `composer update` command.
There are no problems with Symfony 6.4 support

![image](https://github.com/pimcore/pimcore/assets/17727266/efd6e325-1e78-4670-868a-30f74b057af4)

![image](https://github.com/pimcore/pimcore/assets/17727266/07c417b6-386b-49ab-80a5-c7946097471b)

**Comment by fashxp** [comment, marked as answer]
Hi, as @twin-elements already posted, Pimcore requires `^6.2` version of symfony ... which means at the moment it supports 6.2, 6.3 and 6.4 (which is the LTS version of symfony 6). 

BR
Christian

**Comment by azngeek** [comment]
Is there still a roadmap when Pimcore will be updated to support Symfony 7?

**Reply by bluvulture** [reply]
hey @azngeek 

check this one out 

https://github.com/pimcore/pimcore/issues/17696#issuecomment-2422026149
